xray/test/stack-9.0-dependencies.txt

```diff
@@ -126,7 +126,7 @@ integer-gmp 1.1
 integer-logarithms 1.0.3.1
 invariant 0.5.4
 kan-extensions 5.2.3
-lambda-alb 0.0.1
+lambda-alb 0.0.2
```

Comment:
I can't seem to get why this `lambda-alb`  as part of `x-ray` dependencies list.

Reply:
bug in xray devtool setup, does not constrain, misses explicit target.